### ISSUE TITLE
Cannot login to omnivore from the welcome page

### ISSUE THREAD

**Issue by Vishesh-Mistry** [omnivore-app/omnivore#4230]
i use the omnivore chrome browser extension and it asked me to sign in, when i click on login to omnivore i am directed to the login page and when i click on the login option (either sign in with apple or sign in using email) nothing happens i am not redirected to the login page and i cannot right click and find any link assigned to the text as well. i have also uploaded the console output in the browser and I'm using Arc as my browser.
![Capture-2024-07-29-164612](https://github.com/user-attachments/assets/72c6f191-9efd-49a7-bd52-9c424bc15f5d)
<img width="1440" alt="Screenshot 2024-07-29 at 4 49 53 PM" src="https://github.com/user-attachments/assets/c56fe1e4-3988-440f-bc93-3059cf7dbc01">
<img width="529" alt="Screenshot 2024-07-29 at 4 54 14 PM" src="https://github.com/user-attachments/assets/e50d491f-a304-4af8-bbf3-ad43db7b23a1">

**Comment by jacksonh** [comment]
What browser are you using here? The span you have highlighted in the code above has an href on it so should be clickable.

**Comment by Vishesh-Mistry** [comment]
i'm using Arc Browser from the browser company

**Comment by jacksonh** [comment]
Do you maybe have some security software blocking requests?

**Comment by Vishesh-Mistry** [comment]
No, I haven't installed any security software separately, but there may be some software installed in the browser by default.

**Comment by jacksonh** [comment]
We have thousands of logins a day so I suspect this is an extension or something you have installed.

**Comment by princinv** [comment]
same issue here. is Arc browser chromium-based? i have been having an issue with desktop version of Brave all day long. have tried firefox on same device and it worked fine, did a clean install of chromium and it didn't work there either, even with no extensions. 

i am not sure what i would be focusing on to "inspect" and provide additional detail but if someone can point me in the right direction would be happy to post.

**Comment by jacksonh** [comment]
@princinv can you look in Storage -> Cookies then under omnivore.app see if there is one named `auth` ?

**Comment by princinv** [comment]
thanks for getting back so quick @jacksonh 

i wasn't able to see the exact cookies for omnivore.app but i decided to clear them anyway and retry and it worked!

any idea why this would happen both with brave and a brand new install of chromium? while i was troubleshooting a minute ago i tried with google chrome and it worked fine.

thanks again.

**Comment by jacksonh** [comment]
> thanks for getting back so quick @jacksonh
> 
> i wasn't able to see the exact cookies for omnivore.app but i decided to clear them anyway and retry and it worked!
> 
> any idea why this would happen both with brave and a brand new install of chromium? while i was troubleshooting a minute ago i tried with google chrome and it worked fine.
> 
> thanks again.

Can you say what's happening, is it the same thing as above where you just can't click the `Continue with email` button?

**Comment by jacksonh** [comment]
> may have spoken too soon. This is what i see when i login. and if it matters i am using the appleid login method.
> 
> ![image](https://private-user-images.githubusercontent.com/79614477/359767699-549a7825-95f7-4ab5-8859-e1991d5a89d8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQyMjA4NDIsIm5iZiI6MTcyNDIyMDU0MiwicGF0aCI6Ii83OTYxNDQ3Ny8zNTk3Njc2OTktNTQ5YTc4MjUtOTVmNy00YWI1LTg4NTktZTE5OTFkNWE4OWQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIxVDA2MDkwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyOTE4YzY4ZjZiYjQ2NmE4OGI1NWVhZGQ0M2QwYWU2OTRiYWMxMDY5NWZhY2EzYzhiNzQ3YTVmNTJjMWNjNmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.re1A68EZWXuUReCxLIRImoEJI_H5FNrwhS8cw-N9m2I)
> 
> there should be a lot more there (shows up everywhere else). clearing the cookies for omnivore seemed to work for a second but a restarted the browser to check if it still worked and no cigar.

Ah it looks like you are somehow partially logged in here. If you clear site data, like localstorage and cookies, then login again it might work. I think maybe its local storage that wasn't cleared here.

**Comment by princinv** [comment]
do i need to do that for everything? clearing just the site seemed to work briefly but isn't now. maybe i need to clear apple too?

**Comment by jacksonh** [comment]
> do i need to do that for everything? clearing just the site seemed to work briefly but isn't now. maybe i need to clear apple too?

no shouldn't need to clear apple, just the site but need to make sure you clear cookies, session storage, and local storage.

**Comment by princinv** [comment]
> > do i need to do that for everything? clearing just the site seemed to work briefly but isn't now. maybe i need to clear apple too?
> 
> no shouldn't need to clear apple, just the site but need to make sure you clear cookies, session storage, and local storage.

seems to be persisting across relaunches of the browser now. thanks again, hope i dont have to do this regularly.

**Comment by jacksonh** [comment]
> > > do i need to do that for everything? clearing just the site seemed to work briefly but isn't now. maybe i need to clear apple too?
> > 
> > 
> > no shouldn't need to clear apple, just the site but need to make sure you clear cookies, session storage, and local storage.
> 
> seems to be persisting across relaunches of the browser now. thanks again, hope i dont have to do this regularly.

Hopefully not. I think somehow it got into a bad state.

**Comment by jacksonh** [comment]
@Vishesh-Mistry in Arc, if you go directly to https://omnivore.app/auth/email-login can you login?

**Comment by Vishesh-Mistry** [comment]
I'm sorry for the late reply.

I tried to open the url https://omnivore.app/ to sign out of omnivore to check if logging in directly works but the site just keeps loading endlessly.

> @Vishesh-Mistry in Arc, if you go directly to https://omnivore.app/auth/email-login can you login?

I did try opening the above url to login directly and it worked on a different space in the arc browser if you are familiar with arc browser.

I am still new to this so i might have missed some details, but i am happy to help further.

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
